### ISSUE TITLE
3월에 등록한 이슈,마일스톤 등 모든 내용이 삭제됬습니다.

### ISSUE THREAD

**Issue by kangsanKim** [naver/yobi#872]
전원이 나간 뒤 
 YOBI를 다시 실행했더니 
3월에 올린 자료들이 모두 사라졌습니다.
다시 복구할수있는 방법이 있을까요?
빠른 답변 부탁드립니다.

**Comment by changhwa** [comment]
혹시 설정이 메모리DB로 되어있으신가요''?

**Comment by kangsanKim** [comment]
아니요~ 지금까지 잘써왔구요 문제 없었는데 정전으로 인해서 요비가 다운됬었는데 스타트해보니 3월 전체가 보이지 않습니다.

**Comment by doortts** [comment]
안녕하세요?
3월 자료만 사라진건가요?

play가 설치된 위치가 PLAY_HOME 이라고 했을때
예) PLAY_HOME=~/dev/play-2.1.0;export PLAY_HOME

yobi.h2.db 파일이 있는 곳에서 아래 명령어를 실행해 봐 주세요.

```
java -cp $PLAY_HOME/repository/local/com.h2database/h2/1.3.168/jars/h2.jar org.h2.tools.Recover
```
복구 스크립트를 만드는 명령어입니다.
경로 중간에 1.3.168 이 버전인데 혹시 버전이 다르면 이 부분의 이름이 다를 겁니다.
확인해 봐보시고요.

파일을 이용해 DB를 복구하는 방법은
우선 yobi.h2.db 파일을 다른 곳으로 복사해 놓으시고요
```
java -cp h2.jar파일이있는_경로 org.h2.tools.RunScript -url jdbc:h2:file:yobi -user "" -script 앞에서생성된sql파일
```
형태로 실행해 주세요.
